.gitmodules

```diff
@@ -1,11 +1,9 @@
 [submodule "Common/edk2"]
```

Comment:
Please note that upgrading to edk2 introduces a significant amount of effort, as due to the issue with edk2, the Xiaomi 845 is almost unable to start Windows. Therefore, I suggest keeping a usable and stable version of edk2.